conans/client/conan_api.py

```diff
@@ -544,24 +539,67 @@ def package(self, reference="", package_id=None, build_folder=None, source_folde
             self._manager.local_package(package_folder, recipe_folder, build_folder, source_folder)
 
     @api_method
-    def source(self, reference, force=False, cwd=None):
-        cwd = prepare_cwd(cwd)
-        current_path, reference = _get_reference(reference, cwd)
-        self._manager.source(current_path, reference, force)
+    def source(self, path, source_folder=None, build_folder=None):
```

Comment:
Do we want to directly break? Not showing a warning for the ``conan source <pkg_ref>`` command?

Reply:
Look at command to see the check